GeneralsMD/Code/Libraries/Source/WWVegas/WWLib/thread.cpp

```diff
@@ -52,7 +52,7 @@ void __cdecl ThreadClass::Internal_Thread_Function(void* params)
 	tc->running=true;
 	tc->ThreadID = GetCurrentThreadId();
 
-#ifdef _WIN32
+#if defined(_WIN32) && !defined(__GNUC__)
```

Comment:
use `#ifdef _MSCVER` instead if this section is compiler and not platform dependant?

Reply:
I am not sure, use of `__try` and `__except` could [maybe](https://clang.llvm.org/docs/MSVCCompatibility.html) work with clang. (e.g. if clang was used with mingw)